src/libs/connect-lib.js

```diff
@@ -167,5 +159,89 @@ export async function testConnectors(cluster, service, connectors) {
       console.log(`Testing connector: ${connector.name}`);
       return testConnector(workerIp, connector);
     })
+  ).then((res) => {
+    return res;
+  });
```

Comment:
Isn't `.then((res) => { return res; }});` the equivalent of not having the `.then` at all?